integration-tests-ex/cases/src/test/java/org/apache/druid/testsEx/msq/ITMultiStageQuery.java

```diff
@@ -176,4 +188,83 @@ public void testMsqIngestionAndQueryingWithLocalFn() throws Exception
 
     msqHelper.testQueriesFromFile(QUERY_FILE, datasource);
   }
+
+  @Test
+  public void testExport() throws Exception
```

Comment:
Please add an integration test for the access-denied scenario, to ensure that people without the proper EXTERNAL permission cannot do exports.

Reply:
Looking into this.

The new revised integration tests don't seem to enable auth by default,
https://github.com/apache/druid/blob/6d617c34d29509ca41513ed3f964b1430faf5c2f/integration-tests-ex/cases/cluster/Common/environment-configs/auth.env#L18-L20
and there don't seem to be any tests for permissions currently. This might take a little time to setup a role which doesn't have extern permissions for a test for this.